cfgov/jinja2/v1/_includes/molecules/info-unit.html

```diff
@@ -0,0 +1,79 @@
+{# ==========================================================================
+
+   Info Unit
+
+   ==========================================================================
+
+   Description:
+
+   Create an info-unit molecule when given:
+
+   value.modifier:  A modifier class used to alternate the info unit from
+                    stacked to inline layouts
+
+   value.image:     An object containing:
+
+                    url:           The image url
+                    alt:           Image alt
+                    is_square:     Whether the image is square or not.
+                                   Default is false
+                    is_decorative: Whether the image is a bg image or not.
+                                   Default is false
+
+   value.heading:   A string containing title of module.
+
+   value.body:      A string containing the content to display below the
+                    heading (typically a paragraph).
+
+   value.is_button: Boolean if the link should be a button.
+                    Defaut is false
+
+   value.links:     A tuple of items to create a list of links. Objects include
+                    a url and text to display.
+
+                    link_url:  A string for the URL of the link.
+                    link_text: A string for the text of the link.
+
+
+   ========================================================================== #}
+
+{% macro info_unit(value) %}
```

Comment:
You may want to update this to `render` or add a `TODO` to updating the existing codebase. We've used `render` in the past for methods that generate a snippet of HTML, to differentiate from macros that might just update a variable or something, but don't directly render output themselves. I'm not sure how consistently that's used though currently.